doc/source/examples/extended_examples/gui/gui_app.py

```diff
@@ -0,0 +1,236 @@
+import sys
+
+from PySide6 import QtWidgets
```

Comment:
Good catch.

However, are we really running this code? Or are we just offering it for downloading? Because if we dont run it, we don't need to add those dependencies to the ``pyproject.toml`` file :)

Reply:
It doesn't seems that the code is executed. The .py files are just referenced in [executable.rst](https://github.com/ansys/pymapdl/blob/example/gui/doc/source/examples/extended_examples/gui/executable.rst)
For the code in the rst file either I think, because we don't get any output.